server/re/src/reDataObjOpr.cpp

```diff
@@ -1520,7 +1520,9 @@ msiDataObjGet( msParam_t *inpParam1, msParam_t *msKeyValStr,
 /**
  * \fn msiDataObjChksum (msParam_t *inpParam1, msParam_t *msKeyValStr, msParam_t *outParam, ruleExecInfo_t *rei)
  *
- * \brief This microservice calls rsDataObjChksum to chksum the iput data object as part of a workflow execution.
+ * \brief This microservice calls rsDataObjChksum to checksum a data object. Checksum values are
+ *        only calculated on the part of a replica indicated by the size stored in the catalog,
+ *        not necessarily the entirety of the file under management.
```

Comment:
Consider splitting the specifics of the operations from the `\brief` message.
```suggestion
 * \brief This microservice calls rsDataObjChksum to checksum a data object.
 *
 * Checksum values are only calculated on the part of a replica indicated by the size
 * stored in the catalog, not necessarily the entirety of the file under management.
```
Adding the empty line tells doxygen to treat it as the operation details.

Reply:
wasn't sure on the empty line thing.   ignoring for now - looking for other examples.